docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+# Comment Copy Guidelines
+
+This document sets guidelines for the comments produced by the analyzer. The
+[Feedback guidelines][git-feedback-guidelines] talk about _when_ to comment and
+_what_ to comment. This document talks mostly about _how_ to comment.
+
+## Working directory for contents
+
+Comments for the analyzer live in the [`exercism/website-copy`][git-website-copy]
+repository. Changes should be made via a PR on this repository, only changing
+the comments for one analyzer at a time. These will get the
+[`type/analyzer-comments` label][git-website-copy-label].
+
+### Generic comments
+
+Some comments are language specific and not exercise specific. These should be
+extracted into a generic comments file, in order to not duplicate commentary,
+which would lead to either outdated commentary or shotgun surgery when a change
+is made.
```

Comment:
Is there a file structure for generic comment files?

Reply:
Not that I know

Reply:
> a generic comments file

Shouldn't it be "a `general` comments folder`". Both in the `C#` and the `Go` comments we have a `general` folder for general comments useful on multiple or all exercises.

Reply:
Sounds fine with me!

Reply:
Let's make the `$lang.general.xxx` canonical in the docs.